src/assets/stylesheets/base/_variables.sass

```diff
@@ -17,29 +19,35 @@ $m-sm:                            $m-base / 2
 $m-md:                            $m-base / 4 * 3
 $m-lg:                            $m-base * 1.5
 $m-xl:                            $m-base * 2
-$m-xxl:                           $m-base * 2.5
+$m-2xl:                           $m-base * 2.5
+$m-3xl:                           $m-base * 4
 
 //----- border radiuses ------------------------------------------------------------------------------------------------
 $border-radius-base:              $m-base
+$border-radius-md:                $m-xs * 3 / 2
 $border-radius-sm:                $m-xs
 $border-radius-lg:                $m-lg
+$border-size-base:                2px
 
 //----- breakpoints ----------------------------------------------------------------------------------------------------
 $grid-breakpoints:                (xs: 0, sm: 576px, md: 768px, lg: 992px, xl: 1200px)
 $container:                       map-get($grid-breakpoints, xl)
 
 //----- colors ---------------------------------------------------------------------------------------------------------
-$brand-primary:                   #2461bd
+$brand-primary:                   #6d78ed
 $brand-success:                   #48bd66
 $brand-warning:                   #ffd200
 $brand-danger:                    #e62a2a
 $brand-info:                      #5bc0de
 
 $color-black:                     #04080c
-$color-gray-dark:                 hsl(36, 5%, 38%)
-$color-gray:                      hsl(36, 5%, 62%)
-$color-gray-light:                hsl(36, 5%, 75%)
+$color-gray-darkest:              hsl(36, 5%, 20%)
+$color-gray-darker:               hsl(36, 5%, 70%)
+$color-gray:                      hsl(36, 5%, 80%)
```

Comment:
You don't have to change it, but I used to set the names as `dark`, `darker` and `darkest`, so I have 1 more level available just in case - you can't set darkestest :P.  Or you can use the material approach, `color-gray-500` as base and +- 100 for lighter and darker shades.

Reply:
Sure, for the future I think, I'll stick to the material approach, as it seems to be more flexible - I can add more classes before introducing weird variable names